json-rpc/types/src/views.rs

```diff
@@ -192,7 +192,7 @@ pub enum EventDataView {
         time_rotated_seconds: u64,
     },
     #[serde(rename = "createaccount")]
-    CreateAccount {},
+    CreateAccount { created_address: BytesView },
```

Comment:
The Move event looks like

```
struct CreateAccountEvent {
  /// Address of the created account
  created: address,
  /// Role of the created account
  role_id: u64
}
```

I think the Rust event got out of sync (my fault) and we need to add `role_id` here + elsewhere